rnnoise/.eslintrc.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
Have you run `npm run lint`? I saw many format error.

Reply:
Reformatted the code as required and passed the lint.